CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.25.1] - 2024-05-09
+### Added
+- Optional event batching on Event Listener Drivers.
+
+### Changed
+- **BREAKING**: Default behavior of Event Listener Drivers to batch events.
+
```

Comment:
Note that I marked this as a `0.25.1` release but it has a breaking change with goes against semvar. Open to suggestions for how to release this.